docs/index.rst

```diff
@@ -1,49 +1,169 @@
+
+   new_to_edx/mission
+   Our code of conduct <universal_stub>
+   A 15-minute introduction to Open edX systems <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/architecture.html>
+   More about each system <new_to_edx/what_our_systems_do>
+   new_to_edx/data_model
+   new_to_edx/messaging_model
+   What you as a developer can do with the Open edX platform <new_to_edx/what_you_can_do>
+   Recommended setup for development, testing, and deployment <all_development/roadmap/index>
+
+
+Information for all Open edX developers
+=======================================
 
-    Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>
-    developers_guide/index
-    named_releases
-    Open edX Extensions and APIs <https://open.edx.org/extending-edx>
-    (Legacy) Open edX Developer's Guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/>
+You will likely use some or all of the information in this section as you progress
+in your Open edX development work. Take a look through the topics here before you begin
+writing code.
 
 
-Architecture and Best Practices
--------------------------------
+.. From the outline: these topics appear to be of common interest. Are they? i18n, XSS, MFEs (Micro-frontends), IDAs (Independently Deployed Applications, a.k.a. Microservices), Django Configuration/Toggles, Named Releases]
 
 .. list-table::
+   :widths: auto
 
-   * - Open edX Technology Radar (coming...)
-     - A bird's eye view of the decisions, tools and technologies that have been adopted or rejected for Open edX
-       development.
+   * - :doc:`all_development/roadmap/index`
+     - Recommended setup for development, testing, and deployment.
+   * - :doc:`all_development/common_tech/index`
+     - Common elements shared by core platform and extension work.
+   * - :doc:`all_development/contributing/index`
+     - If you want to contribute to Open edX code or documentation, review these topics.
+   * - `Open edX proposals <https://open-edx-proposals.readthedocs.io/en/latest/>`_
+     - Help us decide what comes next and see how we made decisions: our technical proposals.
 
-   * - `Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>`_
-     - Technical decisions made by the Open edX community, in the form of best practices, architecture design,
-       or development processes.
 
-   * - `Architecture and Engineering Confluence page <https://openedx.atlassian.net/wiki/spaces/AC/overview>`_
-     - Confluence page for notes, thoughts, and project-related documents on Open edX architecture and engineering.
+.. toctree::
+   :maxdepth: 4
+   :hidden:
+   :caption: Information for all Open edX developers
+
+   all_development/roadmap/index
+   all_development/common_tech/index
+   all_development/contributing/index
+   Open edX proposals <https://open-edx-proposals.readthedocs.io/en/latest/>
+
+.. Working with the core Open edX platform
+.. =======================================
```

Comment:
What is your intent of keeping this around and commented out? 
My first preference would be for you to remove anything that is unused, and just keep it elsewhere locally (another branch, or backup directory, etc.), rather than reviewing/merging this code.
If this is going to cause you too much pain, I recommend you add a TODO comment with an appropriate explanation, like:
```
.. TODO: due March 2021: Uncomment or delete based on feedback or new content. No need to review until then.
```

Reply:
I'm happy to remove it.

Reply:
All working_with_core material removed: See commit 7e1525f